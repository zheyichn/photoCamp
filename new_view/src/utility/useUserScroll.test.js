import "@testing-library/jest-dom";
// import useUserScroll from "./useUserScroll";
import renderer from "react-test-renderer";
import axios from "axios";
jest.mock("axios");

const fakeReponse = [
  { postId: 1, postContent: "test post one" },
  { postId: 2, postContent: "test post two" },
  { postId: 3, commentText: "test post three" },
];

test("useScroll functions correctly", async () => {
  axios.get.mockResolvedValue({ data: fakeReponse });
  const tree = renderer.create(<useUserScroll></useUserScroll>).toJSON();
  expect(tree).toMatchSnapshot();
});
