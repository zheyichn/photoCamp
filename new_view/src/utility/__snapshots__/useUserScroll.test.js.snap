// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useScroll functions correctly 1`] = `<useUserScroll />`;
